search-app/src/cmr/search/data/complex_to_simple_converters/spatial.clj

```diff
@@ -198,9 +206,12 @@
                          (orbital-condition context shape))
           mbr-cond (br->cond "mbr" (srl/shape->mbr shape))
           lr-cond (br->cond "lr" (srl/shape->lr shape))
-          spatial-script (shape->script-cond shape)
+          operator (spatial-operator (:polygon (:query-params context)))
```

Comment:
So this `ignore-br` option only works with search by `polygon`? What about searching by `bounding_box`, `line`, etc? I don't think this is enough.

Reply:
This search option should be in the form of options that apply to all spatial search parameters, not on `polygon`.

Reply:
I think the code change in spatial plugin supports all spatial types, it is just the way this option is defined in search API that needs to be changed.